lib/rucio/core/replica.py

```diff
@@ -2253,15 +2354,15 @@ def list_and_mark_unlocked_replicas(limit, bytes_=None, rse_id=None, delay_secon
                  models.RSEFileAssociation.name == temp_table_cls.name,
                  models.RSEFileAssociation.rse_id == rse_id)
         ).with_hint(
-            replicas_alias, "index(%(name)s REPLICAS_PK)", 'oracle'
+            replicas_alias, 'INDEX(%(name)s REPLICAS_PK)', 'oracle'
```

Comment:
Unsure the recommended style for this, but should the args of with_hint be on different lines?

Reply:
I placed the args of all `with_hint`s on different lines, like so: 

```
                 models.RSEFileAssociation.rse_id == rse_id)
        ).with_hint(
            replicas_alias,
            'INDEX(%(name)s REPLICAS_PK)',
            'oracle'
        ).outerjoin(
```

Have we decided on a different recommended style for `with_hint`s since you left this comment? If so I am unaware of it, let me know

Reply:
https://rucio.cern.ch/documentation/developer/dev_style_guide#whitespace 

I think we can assume your solution is right?